### DISCUSSION TITLE
listing

### DISCUSSION THREAD

**Discussion by ZRktty** [BetaHuhn/spaces-cli#198] [Q&A] [answered]
Hello!
Is it possible to list contents?
I have files in directories as `<image_id>/image.jpg`

**Comment by BetaHuhn** [comment, marked as answer]
Do you mean listing the contents of a directory in your DigitalOcean Space?

If so, no. `spaces-cli` currently only supports uploading and downloading individual files or whole directories directly, without listing the content of a directory in advance.
